ArduCopter/mode_acro.cpp

```diff
@@ -60,9 +60,9 @@ void ModeAcro::run()
 
     // run attitude controller
     if (g2.acro_options.get() & uint8_t(AcroOptions::RATE_LOOP_ONLY)) {
-        attitude_control->input_rate_bf_roll_pitch_yaw_2(target_roll, target_pitch, target_yaw);
+        attitude_control->input_rate_bf_roll_pitch_yaw_2_cds(target_roll, target_pitch, target_yaw);
```

Comment:
Man, we *really* punted when coming up with this method name for the rate-loop-only stuff!